src/Component/Calculator.js

```diff
@@ -1,52 +1,44 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import View from './View';
 import Button from './Button';
 import calculate from '../logic/calculate';
 
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: '0',
-      next: null,
-      operation: null,
-    };
+const Calculator = () => {
```

Comment:
- [ ] OPTIONAL:  I highly suggest you change your 'view' and 'button' components from class-based to functional with hooks! 💪 That way you will practice and that knowledge will serve you well.  The functional components with hooks are a lot simpler than the class-based ones.